packages/terra-button-group/src/ButtonGroup.jsx

```diff
@@ -87,7 +87,7 @@ class ButtonGroup extends React.Component {
       const cloneChild = React.cloneElement(child, {
         onClick: this.wrapOnClick(child),
         className: cx([{ 'is-selected': isSelected && !child.props.isDisabled }, child.props.className]),
-        'aria-pressed': isSelected || null,
+        'aria-pressed': child.props.isDisabled ? null : isSelected,
```

Comment:
It makes sense that aria-pressed should be true for an active and selected button. If the button is disabled, should aria-pressed be `null` like how it is with the change or should it literally be set to false?

Reply:
Initially I thought it should be false if the button is disabled, but then I found the PR of that line from three years ago: https://github.com/cerner/terra-core/pull/2231#issue-410470597, which indicated we don't want it that way.